vendor/CMakeLists.txt

```diff
@@ -61,3 +61,8 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_FILE_OFFSET_BITS=64")
 # Targets which should be installed by `make install`.
 install(TARGETS adb fastboot "${ANDROID_MKE2FS_NAME}"
 	simg2img img2simg append2simg DESTINATION bin)
+
+# Install bash completion files.
+set(BASH_COMPLETION "${CMAKE_INSTALL_PREFIX}/share/bash-completion/completions")
+install(FILES core/adb/adb.bash RENAME _adb DESTINATION "${BASH_COMPLETION}")
```

Comment:
the source path is `adb/adb.bash` not `core/adb/adb.bash`.

Reply:
also out of curiosity, why the installed filenames start with underscore? like `_adb` and `_fastboot`.

Reply:
> the source path is `adb/adb.bash` not `core/adb/adb.bash`.

Hm?

```
$ git log --format=oneline | head -1
c005b9c54a091f17c525c9e7a65d5afd36d63afc Add patch for missing <limits> include
$ find ./vendor -name adb.bash
./vendor/core/adb/adb.bash
```

> also out of curiosity, why the installed filenames start with underscore?

Whoops, confused that with zsh completion were completions need to be prefixed with an underscore for some reason. Will fix that later.

Reply:
> ```
> $ git log --format=oneline | head -1
> c005b9c54a091f17c525c9e7a65d5afd36d63afc Add patch for missing <limits> include
> $ find ./vendor -name adb.bash
> ./vendor/core/adb/adb.bash
> ```

you are right.

I was testing it with 31.0.0 release and upstream has moved it to the new location https://android.googlesource.com/platform/packages/modules/adb.git/+/refs/tags/platform-tools-31.0.0